docs/Directory.Build.props

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+  <PropertyGroup>
+	  <WasmNativeWorkload>false</WasmNativeWorkload>
+  </PropertyGroup>
+</Project>
```

Comment:
Good question. :) The goal is put this Directory.Build.props file to a root/parent directory where benchmarkdotnet runs from. Due to we [copy docs directory to workitemdirectory](https://github.com/dotnet/runtime/blob/28818b9716fc1e483bb7f6df7ae898751f5a5de3/eng/pipelines/coreclr/templates/run-performance-job.yml#L137), having Directory.Build.props file in the same directory makes it work. However, we should remove ASAP since this could have unexpected back consequences.   opened https://github.com/dotnet/performance/issues/1909 to track this.